### ISSUE TITLE
[Research] OUI Compliance audit of the maps_legacy plugin #4086

### ISSUE THREAD

**Issue by miamia1999** [opensearch-project/OpenSearch-Dashboards#4590]
**Audit**

In the maps_legacy plugin, there are 3 SASS files, namely _leaflet_overrides.scss, _legend.scss and index.scss

 _index.scss_
This SASS file has 2 import statements of the _legend.scss and _leaflet_overrides.scss files. 

__leaflet_overrides.scss_

This is a SCSS (SASS) code block which is primarily used for styling Leaflet map components. Leaflet is an open-source JavaScript library used for interactive maps.
![leaflet_p1](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/d08d727f-346f-491a-bd74-72bf5b636c86)
![leaflet_p2](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/771d2e90-2df4-416d-9e09-8d36192f5be7)
![leaflet_p3](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/61a89583-2f9c-473c-a239-c6194736ef30)

Overall, these styles are used to make the Leaflet map components fit in with the rest of the EUI (Elastic UI) style framework.

 __legend.scss_

This CSS/SCSS code is defining styles for specific elements in a map UI. 

.visMapLegend: 
![legend_p1](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/e7efd577-926d-4e69-a807-76098ccb1345)

.i: 
![legend_p3](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/924e63de-37d3-47bf-9e3c-6d4e14e063a7)

.visMapLegend__title: 
![legend_p4](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/d1f73715-6203-419c-9788-5eb4e355356b)

.visMapLegend__wrapper:
![legend_p2](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/123776561/c92c0eac-a82d-4c8d-a684-ea7ba3816dd9)

In summary, this code is defining styles for a map legend and its components, including the legend title and the legend symbols. The legend is stylized with a specific font, color, padding, and background, and the symbols within the legend are displayed as small, inline circular objects.

**Comment by joshuarrrr** [comment]
Thanks @miamia1999 !
